ogre2/src/media/materials/programs/depth_camera_fs.glsl

```diff
@@ -139,7 +139,8 @@ void main()
   }
 
   // clamp xyz and set rgb to background color
-  if (point.x > far - tolerance)
+  if (point.x * point.x + point.y * point.y >
+      (far - tolerance) * (far - tolerance))
```

Comment:
how about just `length(point) > far - tolerance` to be consistent with how the gpu lidar clips ranges? 

Similarly for clipping points within near clip plane: `length(point) < near + tolerance`

Reply:
Makes sense, I had no idea there was a length function. Good catch

Reply:
Does 9bfe9d1 works?

Reply:
yeah that looks fine. `INTEGRATION_depth_camera` test caught this change and is failing. Can you update the test?

Can you also add a note about this change in the [Migration.md](https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering5/Migration.md) guide?

Reply:
Where should I note the change in the migration guide? 5 -> 6 ?

Reply:
yeah, you can add a new section `Ignition Rendering 5.0 to 6.0`

Reply:
34267e4 should take care of this